diff --git a/node_modules/eris/index.d.ts b/node_modules/eris/index.d.ts
index 8206a36..bd81b3e 100644
--- a/node_modules/eris/index.d.ts
+++ b/node_modules/eris/index.d.ts
@@ -1555,14 +1555,14 @@ declare namespace Eris {
     messageID: string;
     type: Constants["MessageReferenceTypes"]["FORWARD"];
   }
-  interface MessageReferenceReply extends MessageReferenceBase {
+  interface MessageReferenceReply extends Omit<MessageReferenceBase, "type"> {
     messageID: string;
     failIfNotExists?: boolean;
     type?: Constants["MessageReferenceTypes"]["DEFAULT"];
   }
   interface MessageSnapshot {
     guildID?: string;
-    message: Pick<Message, "attachments" | "content" | "edited_timestamp" | "embeds" | "flags" | "id" | "mentions" | "roleMentions" | "timestamp" | "type">;
+    message: Pick<Message, "attachments" | "content" | "editedTimestamp" | "embeds" | "flags" | "id" | "mentions" | "roleMentions" | "timestamp" | "type">;
   }
   interface PartialAttachment {
     description?: string;
